app/client/perf/summary.js

```diff
@@ -0,0 +1,86 @@
+const fs = require("fs");
+const path = require("path");
+
+exports.summaries = async (directory) => {
+  const files = await fs.promises.readdir(directory);
+  const results = {};
+  files.forEach((file) => {
+    if (file.endsWith(".json")) {
+      console.log(file);
+      const content = require(`${APP_ROOT}/traces/reports/${file}`);
+      Object.keys(content).forEach((key) => {
+        if (!results[key]) {
+          results[key] = {};
+        }
+        if (!results[key]?.scripting) {
+          results[key].scripting = [];
+        }
+        results[key].scripting.push(content[key].summary.scripting);
+
+        if (!results[key]?.painting) {
+          results[key].painting = [];
+        }
+        results[key].painting.push(content[key].summary.painting);
+
+        if (!results[key]?.rendering) {
+          results[key].rendering = [];
+        }
+        results[key].rendering.push(content[key].summary.rendering);
+      });
+    }
+  });
+  console.log(results);
+  generateReport(results);
+};
+
+const generateReport = (results) => {
+  var size = 5;
+  Object.keys(results).forEach((key) => {
+    const action = results[key];
+    Object.keys(action).forEach((key) => {
+      size = action[key].length;
+      const sum = action[key].reduce((sum, val) => sum + val, 0);
+      const avg = (sum / action[key].length).toFixed(2);
+      action[key].push(avg);
+    });
+  });
+
+  generateMakdown(results, size);
```

Comment:
Nitpick: Typo in function name. 
```suggestion
  generateMarkdown(results, size);
```